engine/src/history.rs

```diff
@@ -85,19 +86,16 @@ impl History {
                 println!("Couldn't open file because: {}", e);
             }
         }
-        history
+        Ok(history)
     }
 
     pub fn write_move(&self, file_name: &str, turn: usize, board_move: String) {
         let mut file = OpenOptions::new()
             .append(true)
             .open(file_name)
             .expect("game.txt cannot be written to");
-        if let Err(e) = write!(
-            file,
-            "{}. {}\n",
-            turn, board_move
-        ) {
+        if let Err(e) = write!(file, "{}. {}\n", turn, board_move) {
+            //TODO not sure what to do with this one
```

Comment:
imo we just bubble up the error and let the caller (in this case, main) panic. i prefer any panics to occur in an obvious toplevel place like main

Reply:
Oh, this was more in general, because it needs to actually go into the database and not into a file.